README.md

```diff
@@ -51,15 +51,99 @@ Participation in the Kubernetes community is governed by the [Kubernetes Code of
 
 ```bash
 go get github.com/kubernetes-incubator/cri-tools/cmd/critest
-$GOPATH/bin/critest
+go get github.com/kubernetes-incubator/cri-tools/cmd/critctl
 ```
+### critest
 
-### Flags
+ Validation test suites for kubelet CRI
 
-`--runtime-service-address`, `-r`:The default server is dockershim. If we want to test other CRI server such as frakti, we can add flag `--runtime-service-address=/var/run/frakti.sock`. And we can run this test against frakti.
+#### Flags
 
-`--build-dependencies`, `-b`:If we don't need to build dependencies, we can add this flag `--build-dependencies=false` or `-b=false`.
+- `--runtime-endpoint`, `-r`:CRI runtime server endpoint. Default is /var/run/dockershim.sock.
 
-`--ginkgo-flags`,`-g`:Space-separated list of arguments to pass to Ginkgo test runner.
+- `--image-endpoint`, `-i`:CRI server image endpoint, default same as runtime endpoint.
 
-`--focus`,`-f`:CRI e2e test will only run the test that match the focus regular expression.
\ No newline at end of file
+- `--build-dependencies`, `-b`:If we don't need to build dependencies, we can add this flag `--build-dependencies=false` or `-b=false`.
+
+- `--ginkgo-flags`,`-g`:Space-separated list of arguments to pass to Ginkgo test runner.
+
+- `--focus`,`-f`:CRI e2e test will only run the test that match the focus regular expression.
+
+### crictl
+
+CLI for kubelet CRI
+
+- Run podsandbox with config file
```

Comment:
Please move those two examples after Global flags and a header `#### Examples` for them.